src/store/models/app.ts

```diff
@@ -79,6 +79,13 @@ const appModel: AppModel = {
       managed: [],
       custom: [],
     },
+    basePorts: {
+      LND: { grpc: null, rest: null },
+      bitcoind: { rest: null },
+      'c-lightning': { grpc: null, rest: null },
+      eclair: { rest: null },
+      tapd: { grpc: null, rest: null },
```

Comment:
If you set these default values to use the values from [BasePorts](https://github.com/jamaljsr/polar/blob/master/src/utils/constants.ts#L47) then we won't ever need to worry about checking for `null` or `undefined` anywhere in the app.